test/crypto.spec.js

```diff
@@ -241,7 +241,7 @@ test('detectPublicKeyAlgorithms - malformed PEM files should be rejected', t =>
   t.end()
 })
 
-test('detectPublicKeyAlgorithms - public keys should be rejected', t => {
+test('detectPublicKeyAlgorithms - private keys should be rejected', t => {
```

Comment:
This is a typo I noticed. The `detectPublicKeyAlgorithms` function should reject _private_ keys, not _public_ ones.